### PR TITLE
Fix Peertube account subscribers extraction

### PR DESCRIPTION
**Pull request by TiA4f8R**
Fix Peertube account subscribers extraction by adding the subscribers of all channels of an account to the subscribers of this account.

Minimal subscriber count for Framasoft account has been increased to 750 in the test.
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

### CODE REVIEW

**Comment by TobiGr** [comment]
I already took a look at that one, too. But there seems to be a huge difference between the API result for this channel and the follower count [on the website](https://framatube.org/accounts/framasoft): 425 vs 952 
To be honest, I have no idea, why we get different data here.

**Comment by TiA4f8R** [comment]
When I loaded the first time the website, it shows 491 followers first briefly and after 952. I don't understand why.

**Comment by B0pol** [comment]
I noticed something : it is what we've previously called a "parent channel". The channel, owner channel has 491 subscribers, but the subchannels, video channels have 461 subscribers if you sum them all. That makes 952 subscribers

**Comment by TiA4f8R** [comment]
@B0pol I implement what you said, this should be done now.
Tests are failing but it's not related to the subscribers extraction.

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeAccountExtractor.java
Comment (Stypox) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeAccountExtractor.java:82]: Remove this

Comment (TiA4f8R) [reply extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeAccountExtractor.java:82]: I was thinking I remove all prints but no, sorry. Will fix it later.

**Review by Stypox — APPROVED** [review]

**Comment by Stypox** [comment]
@B0pol @TobiGr could you merge this if you think it's ok?

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeAccountExtractor.java
Comment (TobiGr) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeAccountExtractor.java:62]:
```suggestion
        // The subscriber count cannot be retrieved directly. It needs to be calculated.
        // An accounts subscriber count is the number of the channel owner's subscriptions
        // plus the sum of all sub channels subscriptions.
        long subscribersCount = json.getLong("followersCount");
```

**Review by TobiGr — APPROVED** [review]
Looks good to me. We should add a comment about what is done here and why it is done, so other people going through the code will know.

**Comment by TiA4f8R** [comment]
A Peertube test is failing because of a new thumbnail URL. Do not merge this PR, I will fix it soon.

**Pull request merged**
